releasenotes/notes/rename-scenario-var-791bdeaf69a3eeab.yaml

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
Why is this included - it's not an operator-related note, but instead a developer note. This is a test only variable.